apps/dashboard/src/app/(dashboard)/support/create-ticket/components/create-ticket.action.ts

```diff
@@ -98,6 +98,7 @@ export async function createTicketAction(
     loginRedirect("/support");
   }
 
+  // @TODO: This needs to be updated to use team.unthreadCustomerId after all users are migrated.
```

Comment:
This will need to update to use `team.unthreadCustomerId` for the other endpoints to start working.